### PR TITLE
[MIG][12.0] partner_group: Migration to v12.0

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT20168
@carlosdauden @Yajo Can you review?

### CODE REVIEW

File: partner_group/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    'name': 'Partner Group',
+    'summary': 'Group partners by groups (other partner)',
+    'version': '12.0.1.0.0',
+    'development_status': 'Beta',
```
Comment (pedrobaeza) [inline partner_group/__manifest__.py:7]: Put `Production/Stable` and put you as maintainer

File: partner_group/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1,5 @@
+This module extends the functionality of partner to allow group partners into
```
Comment (pedrobaeza) [inline partner_group/readme/DESCRIPTION.rst]:
```suggestion
This module extends the functionality of partner to allow to group partners into
```

File: partner_group/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+other partners.
+This module is an alternative to parent company field, as setting that field
```
Comment (pedrobaeza) [inline partner_group/readme/DESCRIPTION.rst:4]: Leave extra blank line here for new paragraph.

File: partner_group/readme/USAGE.rst [hunk 4]
```diff
@@ -0,0 +1,4 @@
+To use this module you need to:
+
+#. Go to a *Partner*.
```
Comment (pedrobaeza) [inline partner_group/readme/USAGE.rst]: Put a menu path for this (`Contacts` for example).

File: partner_group/readme/USAGE.rst [hunk 5]
```diff
@@ -0,0 +1,4 @@
...
+#. Set group field to other partner.
```
Comment (pedrobaeza) [inline partner_group/readme/USAGE.rst]: Where is set? Which page?

File: partner_group/views/res_partner_views.xml [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <record id="view_partner_form" model="ir.ui.view">
+        <field name="name">Partner Group</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form"/>
+        <field name="arch" type="xml">
+            <field name="lang" position="after">
+                <field name="group_id"/>
```
Comment (pedrobaeza) [inline partner_group/views/res_partner_views.xml:12]: This shouldn't be shown following same logic as accounting info (I think `parent_id` not set).

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Review by yajo — APPROVED** [review]
Code OK.

File: partner_group/views/res_partner_views.xml (hunk 6 above)
Comment (sergio-teruel) [reply partner_group/views/res_partner_views.xml:12]: It has nothing to do with parent_id

**Comment by sergio-teruel** [comment]
Changes done!!

File: partner_group/views/res_partner_views.xml (hunk 6 above)
Comment (pedrobaeza) [reply partner_group/views/res_partner_views.xml:12]: You are not understanding me: this is a commercial field, and thus, it shouldn't be shown to children partners.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**